samples/matter/lock/README.rst

```diff
@@ -560,119 +560,40 @@ Upgrading the device firmware
-
-   #. Click the :guilabel:`Configure` symbol for the ``Time Synchronization`` cluster entry.
-   #. In the **Time Synchronization** context window, go to the **Attributes** tab and enable all required attributes:
-
-      * ``UTCTime``.
-      * ``Granularity``.
-      * ``TimeZone``.
-      * ``DSTOffset``.
-      * ``LocalTime``.
-      * ``TimeZoneDatabase``.
-      * ``TimeZoneListMaxSize``.
-      * ``DSTOffsetListMaxSize``.
-      * ``TrustedTimeSource``.
-
-   #. In the **Time Synchronization** context window, go to the **Attribute Reporting** page and enable all available attribute reporting entries.
-
-   #. In the **Time Synchronization** context window, go to the **Commands** page and enable all required command entries:
-
-      * ``SetUTCTime``.
-      * ``SetTimeZone``.
-      * ``SetTimeZoneResponse``.
-      * ``SetDSTOffset``.
-      * ``SetTrustedTimeSource``.
-
-#. In the **Time Synchronization** context window, go to the **Attributes** tab and set the proper bits for the ``FeatureMap`` attribute:
-
-   * For the ``TimeZone`` support, set the ``0th`` bit of the feature map bit map.
-   * For the ``TimeSyncClient`` support, set the ``3rd`` bit of the feature map bit map.
-
-   As a result, the default decimal value of the ``FeatureMap`` should be `9`.
-
-#. Save the :file:`lock.zap` file, and close ZAP-tool.
-#. Generate new ZAP files with the changes in the Door Lock cluster using the following west command:
+.. _matter_lock_enabling_scheduled_timed_access:
 
-   .. code-block:: console
+Enabling scheduled timed access
+===============================
 
-         west zap-generate
+To enable the scheduled timed access feature use the ``schedules`` snippet available for the :ref:`Matter Lock sample <matter_lock_snippets>`.
+To use the snippet and enable schedule timed access use the ``lock_SNIPPET=schedules`` build flag.
 
-#. Enable the Lock Schedules feature in the |NCS| Matter Lock sample by setting the :kconfig:option:`CONFIG_LOCK_SCHEDULES` Kconfig option to ``y``.
-#. Enable the Read Client support in the |NCS| Matter Lock sample by setting the :kconfig:option:`CONFIG_CHIP_ENABLE_READ_CLIENT` Kconfig option to ``y``.
-#. Use the following Kconfig options to modify the maximum number of specific schedule types:
+For example:
 
-   - :kconfig:option:`CONFIG_LOCK_MAX_WEEKDAY_SCHEDULES_PER_USER` to define the maximum number of ``Week-day`` schedules for one user.
-   - :kconfig:option:`CONFIG_LOCK_MAX_YEARDAY_SCHEDULES_PER_USER` to define the maximum number of ``Year-day`` schedules for one user.
-   - :kconfig:option:`CONFIG_LOCK_MAX_HOLIDAY_SCHEDULES` to define the maximum number of ``Holiday`` schedules.
+.. code-block:: console
 
-To learn more about configuring the Matter clusters, see the :ref:`ug_matter_creating_accessory` user guide.
+   west build -b nrf52840dk/nrf52840 -p -- -Dlock_SNIPPET=schedules
```

Comment:
I think that this part might be redundant as it will be visible under `matter_lock_snippets` after including `.. include:: /includes/sample_snippets.txt` :
```
Specify the corresponding snippet names in the |SNIPPET| CMake option.
```
So IMO the line above might be enough here, thoughts?

Reply:
I've removed it :)